src/lib.rs

```diff
@@ -209,15 +209,25 @@ impl ThreadPool {
         self.panic_count.load(Ordering::Relaxed)
     }
 
-    /// Sets the number of threads to use as `threads`.
+    /// Sets the number of threads to use as `num_threads`.
```

Comment:
Maybe remove this text, so the users of the library just see the deprecation warning.